app/main.py

```diff
@@ -148,20 +149,59 @@ def post(self, serial_number):
                 service,
                 decorator
             )
+            id_blazer = serial_number
+            borrowed_name = ''
+            borrowed_class = ''
+            borrower_contact = ''
+            borrowed_date = ''
+            returned_date = self.request.get('returned')
+            record_values = spreadsheets.get_values(
+                'Records!A2:A',
+                service,
+                decorator
+            )
+            row_1 = len(record_values[0])
```

Comment:
I think this is the glitch. You should be checking the length of the whole array and not just the first element of the array which would be the first column.